authors/apps/authentication/backends.py

```diff
@@ -1,10 +1,58 @@
-# import jwt
-#
-# from django.conf import settings
-#
-# from rest_framework import authentication, exceptions
-#
-# from .models import User
+"""Identify a user using a token."""
+import jwt
+
+from django.conf import settings
+
+from rest_framework import authentication, exceptions
+
+from authors.apps.authentication.models import User
 
 """Configure JWT Here"""
 
+
+class JWTAuthentication(authentication.BaseAuthentication):
+    """Authenticate requests by using tokens."""
+
+    authentication_header_prefix = "Bearer"
+
+    def authenticate(self, request):
+        """Check for authorization header."""
+        request.user = None
+        header = authentication.get_authorization_header(request).split()
+        prefix = self.authentication_header_prefix.lower()
+
+        if not header:
+            return None
+
+        if len(header) == 1:
+            message = "The header requires two paramenters to be passed"
+            return message
+
+        elif len(header) > 2:
+            message = "Token is invalid or provide two paramenters."
+            raise exceptions.AuthenticationFailed(message)
+
+        prefix = header[0].decode('utf-8')
+        token = header[1].decode('utf-8')
+
+        if prefix.lower() != 'Bearer':
+            message = "Bearer prefix missing in authorization header."
+            raise exceptions.AuthenticationFailed(message)
+        return self.authenticate_credentials(request, token)
+
+    def authenticate_credentials(self, request, token):
+        """Identify a user using the token provided"""
+        try:
+            payload = jwt.decode(token, settings.SECRET_KEY, 'utf-8')
+        except BaseException:
+            message = "The token provided can not be decoded."
+            raise exceptions.AuthenticationFailed(message)
+        user = User.objects.get(username=payload['sub']['username'])
+        if not user:
+            message = "User does not exist in the database."
+            raise exceptions.AuthenticationFailed(message)
+        if not user.is_active:
+            message = "User is deactivated."
```

Comment:
Return an appropriate message. eg. 
```suggestion
            message = "User is not active."
```